.github/workflows/ci.yml

```diff
@@ -0,0 +1,95 @@
+name: RSpec CI
+on:
+  push:
+    branches:
+      - 'main'
+      - '*-maintenance'
+      - '*-dev'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  test:
+    name: 'Ruby: ${{ matrix.ruby }}, Rails: ${{ matrix.env.RAILS_VERSION }}'
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+       include:
+         # Rails 6.0 builds >= 2.5.0
+         - ruby: 3.0.0-preview1
+           allow_failure: true
+           env:
+             RAILS_VERSION: '~> 6.0.0'
+         - ruby: 2.7
+           env:
+             RAILS_VERSION: '~> 6.0.0'
+         - ruby: 2.6
+           env:
+             RAILS_VERSION: '~> 6.0.0'
+         - ruby: 2.5
+           env:
+             RAILS_VERSION: '~> 6.0.0'
+         # Rails 5.2 builds >= 2.2.2
+         - ruby: 2.6.6
+           env:
+             RAILS_VERSION: '~> 5.2.0'
+         - ruby: 2.5.8
+           env:
+             RAILS_VERSION: '~> 5.2.0'
+         - ruby: 2.4.10
+           env:
+             RAILS_VERSION: '~> 5.2.0'
+         - ruby: 2.3.8
+           env:
+             RAILS_VERSION: '~> 5.2.0'
+         - ruby: 2.2.10
+           allow_failure: true
+           env:
+             RAILS_VERSION: '~> 5.2.0'
+         - ruby: 2.2.10
+           env:
+             RAILS_VERSION: '5-2-stable'
+         # Rails 5.1 Builds >= 2.2.2
+         - ruby: 2.6.6
+           env:
+             RAILS_VERSION: '~> 5.1.0'
+         - ruby: 2.5.8
+           env:
+             RAILS_VERSION: '~> 5.1.0'
+         - ruby: 2.4.10
+           env:
+             RAILS_VERSION: '~> 5.1.0'
+         - ruby: 2.3.8
+           env:
+             RAILS_VERSION: '~> 5.1.0'
+         - ruby: 2.2.10
+           env:
+             RAILS_VERSION: '~> 5.1.0'
+         # Rails 5.0 Builds >= 2.2.2
+         - ruby: 2.6.6
+           env:
+             RAILS_VERSION: '~> 5.0.0'
+         - ruby: 2.5.8
+           env:
+             RAILS_VERSION: '~> 5.0.0'
+         - ruby: 2.4.10
+           env:
+             RAILS_VERSION: '~> 5.0.0'
+         - ruby: 2.3.8
+           env:
+             RAILS_VERSION: '~> 5.0.0'
+         - ruby: 2.2.10
+           env:
+             RAILS_VERSION: '~> 5.0.0'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: false
```

Comment:
I had weird surprise when it was `true`. Like

<img width="479" alt="Capture d’écran 2020-11-17 à 21 47 57" src="https://user-images.githubusercontent.com/8417720/99880136-dab08600-2c11-11eb-82e0-4f63613537f8.png">

Reply:
😬

Reply:
It's because of the cache name

Reply:
You mean because we use the same ruby version ? 🤔 

I read this https://github.com/ruby/setup-ruby/blob/master/action.yml and didn't find much more information to handle it properly.

Reply:
Can you link a build here https://github.com/ruby/setup-ruby/issues/112